Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/PIDTurn.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc.team199.Robot2018.commands;
+
+import org.usfirst.frc.team199.Robot2018.Robot;
+import org.usfirst.frc.team199.Robot2018.RobotMap;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class PIDTurn extends Command {
```

Comment:
Please see my comments on PIDDrive. They apply here as well. In this case the assymetries will cause the robot to move somewhat forward or backward when attempting to turn in place.